src/stream/src/executor/hash_agg.rs

```diff
@@ -371,11 +419,13 @@ impl<K: HashKey, S: StateStore> HashAggExecutor<K, S> {
         let dirty_cnt = group_change_set.len();
         if dirty_cnt > 0 {
             // Batch commit data.
+            for agg_group in agg_groups.values_mut().flatten() {
```

Comment:
We might be able to move state table commitments to the place near `result_table.commit`, then it's possible to write down register states in `get_output`.

Reply:
> We might be able to move state table commitments to the place near `result_table.commit`, then it's possible to write down register states in `get_output`.

can `get_output` cover?  #5991

Reply:
> We might be able to move state table commitments to the place near `result_table.commit`, then it's possible to write down register states in `get_output`.

Is there any `get_output` which depends on committed data? I think there isn't but I'm not sure.

Reply:
> We might be able to move state table commitments to the place near `result_table.commit`, then it's possible to write down register states in `get_output`.

In my opinion this movement which aims to write down register states in `get_output` will not reduce time and cost (because workload is same), and might make `StateTable` flush delay longer (because some states might cost very long in `get_output`, e.g. 'max')